sdk/main/include/AccountId.h

```diff
@@ -156,9 +156,21 @@ class AccountId
   /**
    * The account ID number.
    */
-  std::optional<uint64_t> mAccountNum;
+  uint64_t mAccountNum;
 };
 
 } // namespace Hedera
 
+namespace std
```

Comment:
There a reason this is in the `std` namespace instead of the `Hedera`?

Reply:
It's my understanding that it's required to be this way to override the standard hash function